elastic-agent.yml

```diff
@@ -17,7 +17,8 @@ outputs:
 
 inputs:
   - type: system/metrics
-
+    # Each input must have a unique ID.
+    id: unique-system-metrics-input
```

Comment:
No ID here only works because it is the only instance of this input. Only when adding a second instance of a `system/metrics` input would this be a problem.